Splitio-net-core-tests/Unit Tests/Events/AsynchronousEventListenerUnitTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Common.Logging;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Splitio.Domain;
+using Splitio.Services.Shared.Classes;
+using Splitio.Services.Shared.Interfaces;
+using System;
+using System.Threading;
+
+namespace Splitio_Tests.Unit_Tests.Events
+{
+    [TestClass]
+    public class AsynchronousEventListenerUnitTests
+    {
+        [TestMethod]
+        public void AddListenerAndPerformLogSuccessfully()
+        {
+            //Arrange
+            var logger = new Mock<ILog>();
```

Comment:
No clue about NET, but isn't there a setUp() method that gets called for each test that we can put duplicate initilaizations?

Reply:
Yes it is, I'll use it here and in the ImpressionListener tests